pkg/containers/containers.go

```diff
@@ -178,7 +178,7 @@ func (c *Containers) CgroupUpdate(cgroupId uint64, path string) (CgroupInfo, err
 }
 
 // extract container id and container runtime from path
-func getContainerIdFromCgroup(cgroupPath string) (string, string) {
+func GetContainerIdFromCgroup(cgroupPath string) (string, string) {
```

Comment:
You can move getContainerID function to this file as well.
Then, you can avoid exporting this function (and any other) and only expose getContainerID, which can be renamed to GetTaskContainerID